t/00_web3_clientVersion.t

```diff
@@ -5,7 +5,7 @@ use Ethereum::RPC::Client;
 
 BEGIN {
     plan skip_all => 'Needs Travis setup'
```

Comment:
```suggestion
    plan skip_all => 'Needs Circle setup'
```

Other tests as well, maybe just CI is better to avoid having to change this in the future as well